pkg/virtctl/create/preference/preference.go

```diff
@@ -24,60 +24,48 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
 	"k8s.io/client-go/tools/clientcmd"
-	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
 	"sigs.k8s.io/yaml"
 
+	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
+
 	"kubevirt.io/kubevirt/pkg/instancetype"
 	"kubevirt.io/kubevirt/pkg/virtctl/create/params"
 )
 
 const (
-	Preference = "preference"
-
 	CPUTopologyFlag        = "cpu-topology"
```

Comment:
nit - What's the justification for exporting flags that are only used in testing and not err strings?

Reply:
Tbh. I'm unsure about it since we don't have explicit rules on that. At least for now it would be consistent with the other files in `virtctl create` though.